server/src/main/java/server/haengdong/application/EventService.java

```diff
@@ -22,4 +23,10 @@ public EventAppResponse saveEvent(EventAppRequest request) {
 
         return EventAppResponse.of(event);
     }
+
+    public EventDetailAppResponse findEvent(String token) {
+        Event event = eventRepository.findByToken(token).orElseThrow(() -> new IllegalArgumentException(""));
```

Comment:
```suggestion
        Event event = eventRepository.findByToken(token)
                .orElseThrow(() -> new IllegalArgumentException(""));
```
c: 한 줄에 하나의 점(.)만 쓰는 것은 어떨까요?